server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailGetMethodContract.scala

```diff
@@ -457,6 +457,60 @@ trait EmailGetMethodContract {
          |}""".stripMargin)
   }
 
+  @Test
+  def convenienceHeaderShouldOnlyTakeFirstHeaderIntoAccount(server: GuiceJamesServer): Unit = {
```

Comment:
I'm not sure but JMAP spec says implementation should match the last header? https://jmap.io/spec-mail.html#properties-of-the-email-object

Reply:
That's unfortunate.

I will have a look at it tomorrow, it will likely involve a lot of boiler plate code with mime4J